ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/controller/PipelineManagementController.java

```diff
@@ -27,7 +27,20 @@ public PipelineManagementController(PipelineManagementService pipelineManagement
 	public List<PipelineTO> overview() {
 		return pipelineManagementService.getPipelines()
 				.stream()
-				.map(config -> fromPipelineConfig(config, pipelineStatusService.getPipelineStatus(config.getName())))
+				.map(config -> fromPipelineConfig(config, pipelineStatusService.getPipelineStatus(config.name())))
 				.toList();
 	}
+
+	@PostMapping()
+	public PipelineTO orchestratorConfig(@RequestBody PipelineConfigTO config) {
+		var pipelineConfig = PipelineConfigTO.fromPipelineConfig(pipelineManagementService.addPipeline(config.toPipelineConfig()));
+
+		return fromPipelineConfig(pipelineConfig, pipelineStatusService.getPipelineStatus(pipelineConfig.name()));
+	}
+
+	@DeleteMapping("/{pipeline}")
+	@ResponseStatus(HttpStatus.OK)
```

Comment:
This is not true at the moment you also give 404 (cfr. my comment in the repository for my thoughts on this)